<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFinancial</class>
 <widget class="QDialog" name="DialogFinancial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogFinancial</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>YEAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCreate">
       <property name="text">
        <string>CREATE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>611</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>CLOSE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogFinancial</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCreate</sender>
   <signal>clicked()</signal>
   <receiver>DialogFinancial</receiver>
   <slot>YearInserted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>YearInserted()</slot>
 </slots>
</ui>
